client/src/main/java/jake2/client/CL.java

```diff
@@ -433,63 +435,6 @@ static void SendKeyEvents() {
         }
     };
 
-    /**
-     * PingServers_f
-     */
-    static Command PingServers_f = (List<String> args) -> {
```

Comment:
Was it just removed?

Reply:
yes, just removed, unnecessary functionality by now